docs/topics/auth/passwords.txt

```diff
@@ -48,12 +48,17 @@ you're not sure, you probably don't.  If you do, please read on:
 
 Django chooses the algorithm to use by consulting the
 :setting:`PASSWORD_HASHERS` setting. This is a list of hashing algorithm
-classes that this Django installation supports. The first entry in this list
-(that is, ``settings.PASSWORD_HASHERS[0]``) will be used to store passwords,
-and all the other entries are valid hashers that can be used to check existing
-passwords.  This means that if you want to use a different algorithm, you'll
-need to modify :setting:`PASSWORD_HASHERS` to list your preferred algorithm
-first in the list.
+classes that this Django installation supports.
+
+For storing passwords, Django will use the first hasher in
+:setting:`PASSWORD_HASHERS`. To store new passwords with a different algorithm,
+modify :setting:`PASSWORD_HASHERS` to list your preferred algorithm first in
+the list.
```

Comment:
`to list ... first in the list` feels a bit mealy, because the latter is referring to the _type_ (ostensibly) or the whole 'sheet' of hashers, while the first `list` is an action to perform - I believe the former/first `list` should be changed to an actual action ... include, set, add, or something along those lines. What're your thoughts?

Reply:
Absolutely, I have changed it to:

```
put your preferred algorithm first in :setting:`PASSWORD_HASHERS`.
```

Reply:
Yep, that works.